### ISSUE TITLE
[Bug]: UI mode results disappear after running suite. Version 1.43

### ISSUE THREAD

**Issue by sakib92** [microsoft/playwright#30519]
### Version

1.43.0

### Steps to reproduce

Run the tests on UI mode.
After the tests are run, the results disappear

https://github.com/microsoft/playwright/assets/7052800/477c6d78-e38b-4716-894c-a76b9b39cd2d

### Expected behavior

The results should remain on the screen

### Actual behavior

The results disappear 

### Additional context

This was not happening on 1.41.0

### Environment

```shell
System:
    OS: macOS 13.6
    CPU: (12) arm64 Apple M2 Max
    Memory: 9.39 GB / 64.00 GB
  Binaries:
    Node: 18.19.1 - ~/.asdf/installs/nodejs/18.19.1/bin/node
    Yarn: 1.22.22 - ~/.asdf/installs/nodejs/18.19.1/bin/yarn
    npm: 10.2.4 - ~/.asdf/plugins/nodejs/shims/npm
    pnpm: 6.32.8 - /opt/homebrew/bin/pnpm
  Languages:
    Bash: 5.2.15 - /opt/homebrew/bin/bash
  npmPackages:
    @playwright/test: 1.43.0 => 1.43.0 
    playwright-slack-report: ^1.1.72 => 1.1.72
```

**Comment by pavelfeldman** [comment]
I believe this has been fixed in 1.43.1, please update.

**Comment by pavelfeldman** [comment]
Closing as per above, please feel free to open a new issue if this does not cover your use case.
